src/components/Card/Card.tsx

```diff
@@ -11,116 +11,225 @@ export interface MyFormValues {
-                    <div className="expDate-field">
-                      <div className="month-field">
-                        <Field className='card-field expDate-input' as="select" name="month">
-                          <option value="">Month</option>
-                          <option value="01">January</option>
-                          <option value="02">February</option>
-                          <option value="03">March</option>
-                          <option value="04">April</option>
-                          <option value="05">May</option>
-                          <option value="06">June</option>
-                          <option value="07">July</option>
-                          <option value="08">August</option>
-                          <option value="09">September</option>
-                          <option value="10">October</option>
-                          <option value="11">November</option>
-                          <option value="12">December</option>
-                        </Field>
-                        <span className={isDarkModeActive ? 'err-msg light-err' : 'err-msg'}><ErrorMessage name="month" /></span>
-                      </div>
+              <label
+                className={
+                  isDarkModeActive
+                    ? "card-field-label light-text"
+                    : "card-field-label dark-text"
+                }
+                htmlFor="cardHolder"
+              >
+                Card Holder
+              </label>
+              <Field
+                className="card-field"
+                name="cardHolder"
+                type="text"
+                data-testid="cardHolderInput"
+              />
+              <span
+                className={isDarkModeActive ? "err-msg light-err" : "err-msg"}
+              >
+                <ErrorMessage name="cardHolder" />
+              </span>
 
-                      <div className="year-field">
-                        <Field className='card-field expDate-input' as="select" name="year">
-                          <option value="">Year</option>
-                          <option value="27">2027</option>
-                          <option value="26">2026</option>
-                          <option value="25">2025</option>
-                          <option value="24">2024</option>
-                          <option value="23">2023</option>
-                          <option value="22">2022</option>
-                        </Field>
-                        <span className={isDarkModeActive ? 'err-msg light-err' : 'err-msg'}><ErrorMessage name="year" /></span>
-                      </div>
+              <div className="expDate-cvv">
+                <div className="expDate-set">
+                  <label
+                    className={
+                      isDarkModeActive
+                        ? "card-field-label light-text"
```

Comment:
there is no need to duplicate card-field-label, it could be better to use string interpolation with nested evaluation